components/telemetry-operator/internal/logger/logger.go

```diff
@@ -29,5 +30,7 @@ func New(format, level string) (*Logger, error) {
 		return nil, err
 	}
 
+	klog.CopyStandardLogTo("ERROR")
```

Comment:
Can you leave a comment what it does and why we need it please? 😁

Reply:
Sometimes logs from controller-runtime are being pushed not to the klog, but to the standard logs, which is happening when some component logs use regular "log" Go package. Because of that, we receive logs in incorrect format. Specifically, it happens when TLS handshake errors occur. Therefore, we need to redirect logs those are being written using standard logging mechanism to our klog. That is basically, why we need the CopyStandardLogTo method. However, we also need to decide to which level we push those logs. Since we consider all logs those are not being logged using klog as unexpected results, we can choose an ERROR as our level. That is why we use "ERROR" as methods' parameter

Reply:
I think the question was about having a comment in the code ;-)

Reply:
Yes, short comment in code please :)